test-solution-template.groovy

```diff
@@ -30,6 +30,7 @@ def DeployJenkinsSolutionTemplate(scenario_name, options) {
     params['storageAccountType'] = ['value' : options.storageType]
     params['publicIPName'] = ['value' : 'jnktst' + UUID.randomUUID().toString().replaceAll('-', '')]
     params['dnsPrefix'] = ['value' : 'jnktst' + UUID.randomUUID().toString().replaceAll('-', '')]
+    params['jenkinsReleaseType'] = ['value' : options.jenkinsReleaseType]
```

Comment:
Oh never mind I see what you're saying. I was confused about the structure of the file/functions